web-app/js/portal/filter/ui/NumberFilterPanel.js

```diff
@@ -115,10 +154,8 @@ Portal.filter.ui.NumberFilterPanel = Ext.extend(Portal.filter.ui.BaseFilterPanel
         });
 
         this._fireAddEvent();
+        trackFiltersUsage('filtersTrackingNumberAction', this._getTrackUsageLabel(), this.layer.name);
```

Comment:
Only issue is here; if I make an operator selection and then add a value in the first field, trackUsage is called twice (once for the operator and one for the numberField selection).